### DISCUSSION TITLE
compiling APS stable version 1.13.5 with new targetSdkVersion

### DISCUSSION THREAD

**Discussion by agrahn** [zeapo/Android-Password-Store#3023] [Q&A] [answered]
I am fine with the feature set of the stable 1.13.5 release of APS. Is it possible to re-compile it from the source to target a more recent Android version, say 13? I installed Android Studio and imported the 1.13.5 source tarball. In file `buildSrc/src/main/java/BaseProjectConfig.kt`, I changed `targetSdkVersion(29)` to `targetSdkVersion(33)`. Then, I re-compiled everything with `./gradlew clean` and `./gradlew assemble` in the terminal. The build finishes successfully, but in all generated `AndroidManifest.xml` files,  `android:targetSdkVersion="29"` still shows up.

What are the steps to upgrade the build for Android 13? Is it possible at all?

**Comment by msfjarvis** [comment, marked as answer]
It should be possible to upgrade it but there are 3 years and 2 thousand commits separating the current development branch and the 1.13.5 release so I can't really help with modifications to that code, you're on your own.

**Reply by agrahn** [reply]
I see, it would require code changes. Nevertheless, thanks for your reply! And thank you for developing APS, which I use a lot.
